src/central-server/management-service/core/src/main/java/org/niis/xroad/cs/management/core/service/ManagementRequestServiceImpl.java

```diff
@@ -96,18 +96,15 @@ private ManagementRequestDto createRequestDto(ClientRequestType request, Managem
         switch (requestType) {
             case CLIENT_REGISTRATION_REQUEST:
                 managementRequest = new ClientRegistrationRequestDto()
-                        .clientId(clientIdConverter.convertId(request.getClient()))
-                        .type(ManagementRequestTypeDto.CLIENT_REGISTRATION_REQUEST);
+                        .clientId(clientIdConverter.convertId(request.getClient()));
```

Comment:
Why type was removed?

Reply:
because type can be resolved from class and jackson gets angry if type is provided also